.ci/setup_env_opensuse.sh

```diff
@@ -15,6 +15,12 @@ source "${cidir}/lib.sh"
 echo "Remove openSUSE cloud repo"
 sudo zypper rr openSUSE-Leap-Cloud-Tools
 
+echo "Add filesystems repo"
+repo_version="15.1"
```

Comment:
I'd consider putting a comment here to explain why and/or how to determine when this needs to be updated to avoid breaking the CI (shudder).

Reply:
changes applied :)